build.gradle

```diff
@@ -20,9 +20,22 @@ checkstyle {
     toolVersion = '10.2'
 }
 
+def pathToFx = System.getenv('PATH_TO_FX')
+
 test {
     useJUnitPlatform()
     finalizedBy jacocoTestReport
+    jvmArgs = [
+            '--module-path', "${pathToFx}",
+            '--add-modules', 'javafx.controls,javafx.fxml'
+    ]
+}
+
+application {
+    applicationDefaultJvmArgs = [
+            '--module-path', "${pathToFx}",
+            '--add-modules', 'javafx.controls,javafx.fxml'
+    ]
 }
```

Comment:
@hjungwoo01 This change is not strictly related to Java 17 If you think this is a useful change, you can send a separate PR for this, with the justifictaion.